subtrack.MAUI/Services/SubscriptionService.cs

```diff
@@ -14,5 +16,25 @@ public async Task<IEnumerable<Subscription>> GetSubscriptions()
         {
             return await _context.Subscriptions.ToListAsync();
         }
+
+        public async Task Update(Subscription subscriptionToUpdate)
+        {
+            var sub = await _context.Subscriptions.FindAsync(subscriptionToUpdate.Id);
+
+            if (sub != null)
```

Comment:
Use defensive coding instead. do the "if sub is null throw the exception" directly, then we dont need an else statement.

same with https://github.com/Code2Gether-Discord/subtrack/pull/44/files